src/SBH_LinearRegressionChart.js

```diff
@@ -149,8 +149,8 @@ class SBH_LinearRegressionChart extends React.Component {
 
 SBH_LinearRegressionChart.propTypes = {
     title: PropTypes.string,
-    scatterData: PropTypes.object.isRequired,
-    lineData: PropTypes.object.isRequired,
+    scatterData: PropTypes.array.isRequired,
+    lineData: PropTypes.array.isRequired,
```

Comment:
How about applying PropTypes.shape for these two props?

Refer to the sample code below.
`
// An object taking on a particular shape
  optionalObjectWithShape: PropTypes.shape({
    color: PropTypes.string,
    fontSize: PropTypes.number
  }),
  
  // An object with warnings on extra properties
  optionalObjectWithStrictShape: PropTypes.exact({
    name: PropTypes.string,
    quantity: PropTypes.number
  }),   
`